packages/frontend/src/components/attacks/Container.vue

```
@@ -2338,6 +2351,14 @@ export default {
                                           {{ finding.severity }}
                                         </span>
 
+                                        <Button
+                                          icon="fas fa-copy"
+                                          size="small"
+                                          severity="secondary"
+                                          @click="copyFindingToClipboard(finding)"
```

ESLint

[error] Replace `copyFindingToClipboard(finding)` with `⏎············································copyFindingToClipboard(finding)⏎··········································`

(prettier/prettier)